src/ConfigGen/Program.cs

```diff
@@ -40,6 +40,7 @@ internal static List<PropertySpecification> RemoveLegacyOrNotRelevant(List<Prope
                 if (p.Name == "request.required.acks") { return false; }
                 // legacy
                 if (p.Name == "consume.callback.max.messages") { return false; }
+                if (p.Name == "offset.store.method") { return false; }

```

Comment:
the incorrect removeDuplicateTopicLevel implementation was by chance omitting this too. not removeDuplicateTopicLevel is fixed, this needs to be added here.